plugins/woocommerce/tests/e2e-pw/test-data/data.js

```diff
@@ -88,11 +513,11 @@ const storeDetails = {
 		},
 		expectedNumberOfIndustries: 7, // There are 7 checkboxes on the page (in Malta), adjust this constant if we change that
 		industries: {
-			other: 'Other',
+			other: getTextForLanguage()['Other'],
 		},
 		products: {
-			physical: 'Physical products',
-			downloadable: 'Downloads',
+			physical: getTextForLanguage()['Physicalproducts'],
```

Comment:
Can we design `getTextForLanguage()` in a way that it will accept the strings directly instead? For example, instead of writing `getTextForLanguage()['Physicalproducts']`, we will only need to write `getTextForLanguage( 'Physicalproducts' )`.
Since the language code will be obtained from the `LANGUAGE` env variable anyway, then there's probably no need to accept a `langCode` parameter in `getTextForLanguage()` unless I'm missing something?